example/package.json

```diff
@@ -15,14 +15,15 @@
     "timeout": "15m"
   },
   "scripts": {
-    "test": "../node_modules/.bin/ava && for x in lambdas/*; do cd $x && npm run test && cd -; done",
-    "int-test": "../node_modules/.bin/jasmine && npm run parallel-tests",
-    "parallel-tests": "sh scripts/tests-parallel.sh",
-    "redeploy-test": "../node_modules/.bin/jasmine spec/standalone/redeployment/*.js",
     "all-tests": "npm run parallel-tests && ../node_modules/.bin/jasmine && npm run redeploy-test",
+    "install-python-deps": "for x in lambdas/*; do cd $x && npm run install-python-deps || true && cd -; done",
```

Comment:
Is this just an alpha reorder?

Reply:
Yeah, sorry, the interesting change is to the `test` and `test:ava` scripts.